common/changes/@boostercloud/application-tester/feat-switch-to-rush_2022-08-18-17-30.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@boostercloud/application-tester",
+      "comment": "Switched to MS Rush",
+      "type": "minor"
+    }
+  ],
+  "packageName": "@boostercloud/application-tester"
+}
```

Comment:
The files in this `common/changes/@boostercloud` folder look really weird to me, are you sure we need them checked in?

Reply:
Yes, these must be checked in because they will be deleted by the publish job. Still, I have to change them so the version bump is to 1.0.0